### PR TITLE
Improve docker support

### PR DESCRIPTION
**Pull request by pladaria**
* removes all .exe wrappers from the repository
* creates .exe wrappers at docker build time

### CODE REVIEW

**Review by Elazul — APPROVED** [review]
nice code factorizing, well done !

**Comment by jespicas** [comment]
@pladaria  I tried your SGDK fork and the branch of this PR and Sonic sample was compiled!! 
Great!

File: readme.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ To build the `sgdk` base image:
 And then to compile the local env, such as `samples` for example:
 
     cd sample/sonic
-    docker run --rm -v $PWD:/src sgdk
+    docker run --rm -v "$PWD":/src sgdk
```
Comment (pladaria) [inline readme.md:85]: otherwise it fails when $PWD has spaces

File: Dockerfile [hunk 2]
```diff
@@ -11,13 +11,17 @@ RUN apk update && \
...
       wine && \
-	  rm -fr /var/cache/apk/*
+        rm -fr /var/cache/apk/*
 
 # Set-up SGDK
 COPY . /sgdk
 ENV GDK=/sgdk
 
+# Create wrappers to execute .exe files using wine
+RUN /sgdk/bin/create-bin-wrappers.sh
```
Comment (pladaria) [inline Dockerfile:21]: Executing this will write the files *inside* the container, keeping the repository clean

**Review by Stephane-D — APPROVED** [review]

**Pull request merged**